framework/include/base/SubProblem.h

```diff
@@ -348,6 +373,7 @@ class SubProblem : public Problem
    */
   void checkMatProps(std::map<unsigned int, std::set<std::string> > & props,
                      std::map<unsigned int, std::multimap<std::string, std::string> > & check_props,
+                     std::map<unsigned int, std::set<MaterialPropertyName> > & zero_props,
```

Comment:
Same here. In general as you see these unsigned int types floating around, let's convert them to the correct types. I've been doing that in `MooseMesh`